lib/samples/add_feature_layers/add_feature_layers_sample.dart

```diff
@@ -60,31 +80,38 @@ class _AddFeatureLayersSampleState extends State<AddFeatureLayersSample> {
   Widget build(BuildContext context) {
     return Scaffold(
       body: SafeArea(
+        top: false,
+        // create a column with a map view and a dropdown button.
         child: Column(
           children: [
+            // add a map view to the widget tree and set a controller.
             Expanded(
               child: ArcGISMapView(
                 controllerProvider: () => _mapViewController,
+                onMapViewReady: _onMapViewReady,
               ),
             ),
-            DropdownButton<String>(
+            // create a dropdown button to select a feature layer source.
+            DropdownButton(
               alignment: Alignment.center,
               hint: const Text(
                 'Select a feature layer source',
                 style: TextStyle(
                   color: Colors.deepPurple,
                 ),
               ),
+              // set the selected feature layer source.
               value: _selectedFeatureLayerSource,
               icon: const Icon(
                 Icons.arrow_drop_down,
                 color: Colors.deepPurple,
               ),
               elevation: 16,
               style: const TextStyle(color: Colors.deepPurple),
-              onChanged: (String? featureLayerSource) {
+              // set the onChanged callback to update the selected feature layer source.
+              onChanged: (Source? featureLayerSource) {
```

Comment:
I think we don't need `Source?` here -- it can just be `onChanged: (featureLayerSource) {`